twisted/twisted
# [8730] Send per-job status for Travis.

Scope
=====

This updates the travis-ci build system to send separate commit status for each Travis-CI job.

In this way, when travis fails, you can go directly to the job.

RIght now, you have to go to the build, check the list of jobs and then click on the ones which are red.

See: https://twistedmatrix.com/trac/ticket/8730


Changes
=======


Add travis before after-success and after-failure hooks to update the commit status.


How to test
=========

See the commit status for this PR.

Leave your feedback if you find this an improvement, no improvement or an annoyance.

The current version has conflicts, but is not reviewed since 2016 so at this point there is not much use to keep it up to date with trunk

Thanks
